lein-cljfmt/src/leiningen/cljfmt.clj

```diff
@@ -8,8 +8,12 @@
             [leiningen.cljfmt.diff :as diff]
             [meta-merge.core :refer [meta-merge]]))
 
+(defn remove-dir-from-filename [dir filename]
+  (str/replace filename dir ""))
```

Comment:
This doesn't ensure the `dir` is removed from the beginning of the string only, right?

Perhaps something more like:

```clojure
(if (.startsWith ^String filename dir)
  (subs filename (count dir))
  filename)
```